src/components/common/Members/Members.tsx

```diff
@@ -0,0 +1,204 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { ColonyRole } from '@colony/colony-js';
+import sortBy from 'lodash/sortBy';
```

Comment:
On master we have the pattern of importing to `utils/lodash` and then reimporting from there. Makes it nicer to import lodash modules if you need more than one in the same file.